lib/bolt/cli.rb

```diff
@@ -212,6 +214,11 @@ def print_help(mode)
       puts parser.help
     end
 
+    def parse_nodes(nodes)
+      list = get_arg_input(nodes)
+      list.split(/[[:space:],]+/).map(&:strip).reject(&:empty?)
```

Comment:
this strip should be unnecessary with this regex

Reply:
IIRC the split doesn’t handle leading spaces for the first entry or trailing spaces for the last entry. I could probably strip before splitting though.

Reply:
nah, you're right, `strip` isn't needed.